### ISSUE TITLE
ID 260767: [Programmatic Access - Simon - Alert rule>Search by data source]: Visual label and aria label is not same for 'Search by data source' combo box under "Alerting" screen.

### ISSUE THREAD

**Issue by eledobleefe** [grafana/grafana#65845]
### **User Experience:** 
Visually impaired users who depend on screen readers will get disoriented and will face difficulty in understanding the control if `aria-label` and visual label do not match each other. 
_Note: User credentials should NOT be included in the bug._

### **Repro Steps:**

1. Open #URL: [play.grafana.org](https://play.grafana.org/).  
2. Sign in with valid credentials. "Home" page appears.  
3. Navigate to the left navigation bar and to the "Alerting" button, select it.  
4. It gets expanded and navigates the expanded sub menu items. Select the "Alert Rules" menu items. 
5. New page appears. Navigate the tab which opens.  
6. Navigate to "Search by data Sources" field and select it.

**Actual Result:** 
Issue: Visual `label` and `aria label` is not same for 'Search by data source' combo box under "Alerting" screen.

**Observation:** 
Screen reader is reading as "Data source picker select container group Select a data source combo edit" when focus moves on 'Search by data source' combo box.

**Expected Result:**
Visual label and aria label should be same for 'Search by data source' combo box under "Alerting" screen i.e., "Search by data source"
_Note:  The same issue exists with NVDA and JAWS. Same issue exists with alerting->Alerting groups_

### **Severity:** 
`SEV3`

### **Area:**
`DataSourcePicker component`

**Comment by grafanabot** [comment]
@eledobleefe please add one or more appropriate labels. Here are some tips:

- if you are making an issue, TODO, or reminder for yourself or your team, please add one label that best describes the product or feature area. Please also add the issue to your project board. :rocket:

- if you are making an issue for any other reason (docs typo, you found a bug, etc), please add at least one label that best describes the product or feature that you are discussing (e.g. `area/alerting`, `datasource/loki`, `type/docs`, `type/bug`, etc). [Our issue triage](https://github.com/grafana/grafana/blob/main/ISSUE_TRIAGE.md#3-categorizing-an-issue) doc also provides additional guidance on labeling. :rocket:

 Thank you! :heart:

**Comment by ckbedwell** [comment]
This issue appears to be resolved as of Grafana 10.2.0.
